### PR TITLE
Added an advanced example, returning a different filename.

### PR DESCRIPTION
**Pull request by clalarco**
I added an advanced example, where a different filename is returned. This worked for me and it's not documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,21 @@ Use the `igorw_file_serve.response_factory` service to create a FileServe respon
...
+    $real_path = '../VERSION';
+    $returned_filename = 'my_filename.txt';
+    
+    $file_options = array( 'serve_filename' => $returned_filename,
+                         );
+
+    $response = $this->get('igorw_file_serve.response_factory')
+                ->create( $filename = $real_path, 
+                          $contentType = 'application/octet-stream',
+                          $options = $file_options,
```
Comment (stof) [inline README.md]: why assigning new variables here ?

Comment (clalarco) [inline README.md]: To clarify the example code. If you like you can write it in a line, but for me it's a bit confusing.
Now that including the arguments names have the same intention.

Comment (stof) [inline README.md]: the issue is not about using several lines but about creating the `$filename`, `$contentType` and `$options` variables (btw, I'm not even sure it is valid)

**Comment by clalarco** [comment]
However, this worked fine.

**Comment by igorw** [comment]
Merged, thanks.

**Pull request merged**